cl/recap/api_serializers.py

```diff
@@ -148,6 +150,12 @@ def validate(self, attrs):
                     "document_number fields completed."
                 )
 
+            if not re.match("^[0-9]*$", attrs.get("document_number")):
```

Comment:
Hm. I think this might soon be false. @ERosendo in ACMS, do we wind up with `document_number`'s that have dashes?

Reply:
@mlissner In ACMS, the `document_number` consistently appears as an integer value, while the `pacer_doc_id` is the one that includes dashes.

Here's an example:

<img width="969" alt="Screenshot 2023-11-20 at 8 46 37 PM" src="https://github.com/freelawproject/courtlistener/assets/55959657/0a91b3cf-2501-4646-bbf2-d66f3cb4104a">

From that download confirmation page, the extension will extract the following data:

```
document_number: 18
pacer_doc_id: c3324852-8d2c-ee11-9965-001dd83014ad
```

Reply:
I'll get with the ACMS program eventually, sorry for being slow. Thanks for the response.

Reply:
No worries! I'm happy to help 😃

Reply:
I'm trying not to be helpless. :)